src/install.rst

```diff
@@ -270,11 +273,19 @@ These instructions will install the Nextstrain CLI and tools to run and view you
                      # follow through installation prompts
                      rm get-docker.sh
 
+                  .. note::
+
+                     Alternatively, you can install Docker Engine using your system's package manager:
+
+                     .. code-block:: bash
+
+                        sudo apt install docker.io
```

Comment:
Cool, wasn't aware of this option since it wasn't advertised on https://docs.docker.com/engine/install/ubuntu/. Any reason not to have this as the only option to keep things simple? With a more general note like:

> See [Docker's installation documentation](https://docs.docker.com/engine/install/ubuntu/) for alternative installation methods.

Reply:
Oh wait, that page actually says:

> Older versions of Docker were called `docker`, `docker.io`, or `docker-engine`. If these are installed, uninstall them

So maybe we shouldn't advertise this method?

Reply:
`docker.io` is the package provided by the Ubuntu/Debian maintainers.

`docker-ce` and co. are the packages provided by the Docker maintainers.

The latter will generally be the latest versions of Docker while the former will lag per Ubuntu/Debian's typical stability policies. OTOH, the latter requires setup of Docker's third-party package repos, signing keys, etc. while the former does not. Since we don't need the latest and greatest version, it seems simpler to use the Ubuntu-provided packages. Docker instructs you to uninstall `docker.io`, etc. because you can't have both Ubuntu's package and Docker's package installed, and `apt` will report a conflict if you try to.

Reply:
All that said, I'd be in favor of making the simplest option the most prominent as you initially suggested.

Reply:
Repushed, replacing a548356 with 21aa2a4.